src/Symfony/Component/Translation/Dumper/JsonFileDumper.php

```diff
@@ -23,9 +23,23 @@ class JsonFileDumper extends FileDumper
     /**
      * {@inheritdoc}
      */
+    protected function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
+    {
+        $flags = isset($options['json_encoding']) ? $options['json_encoding'] : 0;
+
+        if (defined('JSON_PRETTY_PRINT')) {
+            $flags |= JSON_PRETTY_PRINT;
```

Comment:
should we really force it all them time, even when encoding options are specified ? this forbids disabling the pretty printing